clusters/build-clusters/build-shared/supplemental-ci-images/cli-yq_mabc.yaml

```
@@ -4,34 +4,33 @@ metadata:
             curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_${ARCH} -o /tmp/yq && chmod +x /tmp/yq
         RUN yum install -y jq
       type: Dockerfile
     strategy:
       dockerStrategy:
         from:
           kind: DockerImage
-          name: quay.io/openshift/ci:ocp_4.18_cli
+          name: registry.access.redhat.com/ubi9/ubi
         imageOptimizationPolicy: SkipLayers
       type: Docker
+    successfulBuildsHistoryLimit: 1
```

YAMLlint (1.37.1)

[error] duplication of key "successfulBuildsHistoryLimit" in mapping

(key-duplicates)